### ISSUE TITLE
Text placeholder not being updated in translated notifications

### ISSUE THREAD

**Issue by RuiPereira** [jitsi/jitsi-meet#4667]
## Description
---
I'm running jitsi-meet on Debian Stretch installed from the stable binary packages, with default langage set to French (`defaultLanguage: 'fr'.`)

After updating to the most recent versions (dating end of August) there a bunch of text placeholders (like `__room__`) that are no longer replaced properly in the notifications of the application, eg.:

![image](https://user-images.githubusercontent.com/349989/65418100-f8127600-ddfb-11e9-89d7-a078dd9419df.png)

## Current behavior
---
The placeholders appear even after cleaning all the cached files in the browser. Everything works fine if the default language is set to English.

# Environment details
---
jitsi-meet                            1.0.3936-1                   
jitsi-meet-prosody                    1.0.3577-1                    
jitsi-meet-web                        1.0.3577-1                   
jitsi-meet-web-config                 1.0.3577-1                           
jitsi-videobridge                     1124-1                                     
jicofo                                1.0-481-1

**Comment by damencho** [comment]
Can you easily check whether you see those when using latest from unstable?

**Comment by RuiPereira** [comment]
That's a production server, so I can't check easily. I'll try to reproduce the problem on a test server.

**Comment by RuiPereira** [comment]
> Can you easily check whether you see those when using latest from unstable?

I was able to reproduce the issue on a test server using `stable`. It disappeared once I updated the packages to `unstable`. So, the question is, how *unstable* is `unstable`? 😄 Can I install those on a production server, or should I wait until the next stable release?

Thanks.

**Comment by damencho** [comment]
So the packages in unstable a pretty stable I would say :) we try to keep them like that. This is what we deploy on meet.jit.si, even before pushing to stable. I will check with the team today what is the state, and I'm pretty sure we can push to stable what we have on meet.jit.si right now.

**Comment by RuiPereira** [comment]
Hi. Just for the record, I upgraded to `unstable` (no config changes) and everything seems to be working, except `jigasi` (1.1-29) which is throwing the following error when a phone call is transferred to the room

`2019-09-27 09:58:09.102 SEVERE: [43] impl.protocol.sip.SipLogger.logError().133 Error from the JAIN-SIP stack: Invalid argument address = ::0 port = 5060 transport = UDP`

The call is then refused. Possibly some config error? Anyway, I downgraded `jigasi` back to 1.0-245 and it's working again.

**Comment by damencho** [comment]
I'm interested in seeing logs of jigasi not working, so we can work on fixing it. If you have logs from a not working session, please send them.

**Comment by RuiPereira** [comment]
Here's one (IPs redacted) : [jigasi.log](https://github.com/jitsi/jitsi-meet/files/3661950/jigasi.log)

**Comment by damencho** [comment]
I see:
```
2019-09-27 10:00:17.566 INFO: [118] org.jitsi.jigasi.SipGatewaySession.handleCallState().1084 [ctx=15695712158481172937799] SIP call ended: CallPeerChangeEvent: type=CallPeerStatusChange oldV=net.java.sip.communicator.service.protocol.CallPeerState:Connecting newV=net.java.sip.communicator.service.protocol.CallPeerState:Disconnected for peer=007XXXXX <007XXXXXXX@XX.XX.XX.XX>;status=Disconnected
2019-09-27 10:00:17.572 INFO: [118] org.jitsi.jigasi.SipGatewaySession.peerStateChanged().1150 [ctx=15695712158481172937799] SIP peer state: Disconnected

```
The SIP peer got disconnected immediately after connecting. Hum, can you send me and the pcaps logs of such a session. You can send it privately to damencho AT jitsi dot org, as there will be a lot of addresses and stuff in it.

**Comment by RuiPereira** [comment]
Sent, thanks.

**Comment by RuiPereira** [comment]
Hello. For the record I managed to fix the problem with jigasi 1.1 using [this](https://community.jitsi.org/t/unexpected-behavior-when-jigasi-update-from-1-0-to-1-1/19684/10) info.
